Dockerfile

```diff
@@ -15,16 +15,15 @@ RUN mkdir /home/raku \
     && git clone https://github.com/tadzik/rakudobrew ~/.rakudobrew \
     && echo 'eval "$(~/.rakudobrew/bin/rakudobrew init Sh)"' >> ~/.profile \
```

Comment:
Since this is inside ".profile", it's going to use this when opening a shell. This needs to be eliminated too.

Reply:
Done